internal/services/controller/handlers/filter.go

```diff
@@ -0,0 +1,19 @@
+package handlers
+
+import (
+	"castai-agent/internal/castai"
+)
+
+type Filter func(e castai.EventType, obj interface{}) bool
```

Comment:
do we have any filters define or is just a boilerplate for now? from code perspective only "transformers" exist

Reply:
We do not, but I need this to for my following PR, where I add filters for k8s events, because for now we're only interested in pod OOM events.